types/react-toastr/index.d.ts

```diff
@@ -10,10 +10,10 @@ export class ToastContainer extends Component<{
     toastMessageFactory: any;
     className?: string;
 }> {
-    error: (message: string, title: string, optionsOverride?: {}) => void;
-    info: (message: string, title: string, optionsOverride?: {}) => void;
-    success: (message: string, title: string, optionsOverride?: {}) => void;
-    warning: (message: string, title: string, optionsOverride?: {}) => void;
+    error: (message: any, title: any, optionsOverride?: any) => void;
```

Comment:
If the intent is a React element, you should use `React.ReactNode` instead of `any`.

Is there a reason `optionsOverride` was changed from a generic object to `any`?